src/components/Features/Features.jsx

```diff
@@ -0,0 +1,173 @@
+import * as styles from './Features.module.scss';
+
+import React, { useState } from 'react';
+import {
+  collapsableList,
+  customerIcons,
+  featuresList,
+  frameworkIconsDotNet,
+  frameworkIconsJava,
+  frameworkIconsJavascript,
+  frameworkIconsOther,
+  frameworkIconsPhp,
+  frameworkIconsPython,
+  languageList,
+  navigationList,
+} from './dataSource';
+
```

Comment:
Lets keep consistency order of imports
```suggestion
import React, { useState } from 'react';

import * as styles from './Features.module.scss';

import {
  collapsableList,
  customerIcons,
  featuresList,
  frameworkIconsDotNet,
  frameworkIconsJava,
  frameworkIconsJavascript,
  frameworkIconsOther,
  frameworkIconsPhp,
  frameworkIconsPython,
  languageList,
  navigationList,
} from './dataSource';

```

Reply:
Resolved